src/node.rs

```diff
@@ -35,12 +41,21 @@ pub struct Node {
     /// Specify the network of this node.
     #[structopt(default_value = "2", short = "n", long = "network")]
     pub network: u16,
+    /// The listener IP of the node.
+    #[structopt(default_value = "0:0:0:0", long = "ip")]
+    pub ip: String,
     /// Specify the port for the node server.
-    #[structopt(long = "node")]
-    pub node: Option<u16>,
+    #[structopt(long = "listener-port")]
+    pub listener_port: Option<u16>,
+    /// Disable the RPC server.
+    #[structopt(long = "disable-rpc")]
+    pub disable_rpc: bool,
+    /// The IP address of the RPC server.
+    #[structopt(default_value = "0:0:0:0", long = "rpc-ip")]
+    pub rpc_ip: String,
```

Comment:
Why is this necessary?

Reply:
See https://github.com/AleoHQ/snarkOS/pull/1286#discussion_r750972245.

Reply:
This should also be an `IpAddr`?

Reply:
I didn't think `structopt` was able to interpret the default value "on the fly" like that, but it appears that it is :+1:.